scripts/mobile-cv-suite-config.cmake

```diff
@@ -80,12 +80,13 @@ if (ANDROID)
     z) # zlib, required by OpenCV stuff
 else()
   if (_MCS_USE_SLAM)
-    list(APPEND _MCS_INTERFACE_LIBS ${_MCS_LIBS}/libpangolin.so)
+    find_package(Pangolin REQUIRED PATHS "${_MCS_LIBS}/cmake/Pangolin" NO_DEFAULT_PATH)
```

Comment:
Did you try this

    list(APPEND _MCS_INTERFACE_LIBS ${_MCS_LIBS}/libpangolin${CMAKE_SHARED_LIBRARY_SUFFIX})

instead of `find_package`?

Reply:
I had that at first, but at least on Mac it fails to find the GL/glew.h header file. Using the find_package (I think) triggers the cmake modules from Pangolin that finds it and gives it to the compiler&linker